docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites.md

```diff
@@ -42,6 +34,20 @@ A node running YugabyteDB Anywhere is expected to meet the following requirement
 - 100 GB SSD disk or more
 - 64-bit CPU architecture
 
+### Kubernetes-based installations
+
+A Kubernetes node is expected to meet the following requirements: 
+
+- 4 cores (minimum) or 8 cores (recommended)
```

Comment:
The minimum should be 5 cores. We request 4.75 cores in YBA k8s installation.

Reply:
Changed